example.env

```
@@ -1,27 +1,37 @@
 # variables for conduit-backend
 
 # essential cryptographic key used by Django to protect sensitive data and provide security-critical functionality
-SECRET_KEY=
+SECRET_KEY=2^f+4@v7$v1f9yt0!s)2-1t$)tlp+&m17=*g))_xoi&&9m#2ax
```

```suggestion
SECRET_KEY=secret
```

for better readability